lib/disk/modules/VMWareSparseDisk.rb

```diff
@@ -161,6 +262,24 @@ def allocGrain(gn)
     sector * blockSize
   end
 
+  def parseGrainTable(buffer)
+    gt = []
+    (0..(GTES_PER_GT - 1)).each do |index|
```

Comment:
I think this whole method can be

```ruby
GTS_PER_GT.times.map { |index| buffer[index, GTE_SIZE].unpack1('L') }
```

Reply:
Thanks for the suggestion.  Implemented.